package.json

```diff
@@ -30,5 +30,8 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "dependencies": {
+    "cypress": "^13.15.0"
```

Comment:
O Cypress deveria ter sido instalado como `devDependencies`, não `dependencies`.

Segue uma explicação detalhada abaixo:

# Instalação de libs (ou packages) como `dependencies` em vez de `devDependencies`

É comum alunos/as instalarem packages do npm que servem para suportar o desenvolvimento de aplicações como `dependencies` em vez de `devDependencies`.

## Por que tal prática fere um bom design de testes?

Tal prática prejudica o desempenho do projeto, visto que mais pacotes externos exigem mais tempo para serem baixados da internet e em esteiras de entrega contínua, é possível configurar que somente as dependencies sejam instaladas no momento da implantação de uma nova versão em um ambiente produtivo, por exemplo, sem perda de tempo instalando `devDependencies`.

## Sugestão com um melhor design

Basta entender o pra que serve cada pacote e instalar os pacotes certos do jeito certo.

Aqui vai uma breve explicação:

No contexto de módulos npm, a principal diferença entre `dependencies` e `devDependencies` está no momento em que os pacotes são necessários:

- `dependencies`: São os pacotes essenciais que o projeto precisa para funcionar em produção. Esses módulos são instalados quando o projeto é executado em ambientes finais, como servidores de produção ou em builds de lançamento. São usados diretamente pelo código do projeto durante a execução (ex.: React).
- `devDependencies`: São pacotes usados apenas durante o desenvolvimento e testes do projeto. Incluem ferramentas como linters, frameworks de testes e geradores de documentação. Esses módulos não são necessários em produção e, portanto, não são instalados quando o projeto é implementado nesse ambiente, exceto se forem explicitamente requisitados.
- 
Em resumo, `dependencies` são para a execução do projeto, enquanto `devDependencies` são para o suporte ao desenvolvimento do mesmo.

Reply:
Faltou esse ajuste.

Reply:
Ok.   },
  "devDependencies": {
    "@babel/plugin-proposal-private-property-in-object": "^7.21.11",
     "cypress"